package.json

```diff
@@ -16,18 +17,24 @@
     "build:dev": "rollup -c --watch",
     "docs": "npm run build && vuepress build docs --no-cache",
     "docs:dev": "npm run build && vuepress dev docs --no-cache",
-    "lint": "eslint . --ext .js",
-    "package": "npm run build && node scripts/create-packages.js"
+    "lint": "eslint . --ext .js,.ts --cache",
+    "package": "npm run build && node scripts/create-packages.js",
+    "test": "npm run test-types",
+    "test-types": "tsc -p types/test"
   },
   "devDependencies": {
     "@simonbrunel/vuepress-plugin-versions": "^0.2.0",
+    "@typescript-eslint/eslint-plugin": "^5.10.1",
+    "@typescript-eslint/parser": "^5.10.1",
     "@vuepress/plugin-google-analytics": "^1.9.5",
     "archiver": "^5.3.0",
     "chart.js": "^3.7.0",
     "eslint": "^8.5.0",
     "eslint-config-chartjs": "^0.3.0",
+    "eslint-plugin-es": "^4.1.0",
```

Comment:
I don't think this package is necessary as it is not used in any ESLint configuration file. For the datalabels plugin, it's used in [this config](https://github.com/chartjs/chartjs-plugin-datalabels/blob/master/src/.eslintrc.yml#L1). However, I wouldn't make this change as part of this PR, should we ever need to.

Reply:
Yeah, mb, got an error from missing plugin so added all of them from datalabels but I got version mismatch and forgot to remove it but fixed it now 👍